### PR TITLE
[SCH-263] HD video quality alert

### PR DESCRIPTION
**Pull request by ivanjiang5628**
## Description
- [Linear Ticket](https://linear.app/jane/issue/SCH-263/telehealth-hd-video-quality-alert)
- [Slack](https://janeapp.slack.com/archives/CTQRM510Q/p1614973707006800)

Since we want to prompt practitioners to enable the HD video beta feature in Jitsi only if they have a good connection. This PR updates the HD video loading flow as below:

Screen Shot 2021-03-05 at 11 38 30 AM

- Adds an HD video alert component.
- Adds a new HD video constrain setting to keep the HD video full screen since we've added padding to the SD video.
- Limit the video quality to HD(360) when initiating the call.
- Detects the actual connection quality strength in `_onStatsUpdated` listener and shows the `HD video quality alert` only if the practitioner has a "good" connection (connection quality strength > 30, bandwidth bitrate > 2500kps).
- Only allows the practitioner to control the resolution/video quality. 
- Hides the video quality control option for patients.

### General PR Class
👁 = UX / UI improvement
🌟 = New Feature (Adds Functionality) 

### Dependencies / ENV
N/A

- [ ] requires env configuration to be added in production
- [ ] js package changes<sup>1</sup>
- [ ] more than 200 LOC changed in production files<sup>1</sup>
- [ ] includes a user-facing workflow change to an existing production feature (user muscle memory or pattern recognition will be affected)
- [ ] could prevent access to Jane Video (eg. cors, middleware, changes to auth system)
- [ ] affects a widely used component or piece of code
- [ ] I have a doubt - I want the RMT to review this. If possible, please elaborate your concerns in the risk assessment section.

<sup>1</sup> No need to explain these risk factors below

### Release Risk Assessment
Low

### Demo Notes
Loom Demo Video:
https://www.loom.com/share/2f9f771db8e947df9ef03f73b68e55e3

## Code Review
Resource: [Dev Team Notion Page](https://www.notion.so/janeapp/Dev-Team-f06c6eb2ccca4066bc63fc1ac1bd2549)
Resource: [Code Review Checklist](https://www.notion.so/janeapp/Code-Review-checklist-2c510c527ac7470c902a5e8f25f9db3c)

- [x] I clearly explained the WHY behind the work, in the Description above

#### Design
- [x] I added instructions for how to test, in the QA section below
- [x] I added specs for changes, or determined that none were required
- [ ] I demoed this to the appropriate person
- [x] I considered both mobile & desktop views, or that wasn't relevant

#### Code
- [x] I committed code with informative git messages
- [x] I wrote readable code, or added comments if it was complex
- [x] I performed a self-review of my own code
- [x] I rebased my branch on the latest `master`

### Steps to Reproduce
Please make sure the `Telehealth Selectable HD quality` beta flag is enabled > Create an online appointment > join the call > wait until the `Load HD for better video quality?` alert pops up. (if the connection is "good" & the bandwidth > 2000) > click on the `Improve my video quality` button > the video feed should change to HD(720) at full screen.

### Fixed / Expected Behaviour
Loom Demo Video:
https://www.loom.com/share/2f9f771db8e947df9ef03f73b68e55e3

Hd video quality alert component:

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -103,5 +103,6 @@ $flagsImagePath: "../images/";
 @import 'e2ee';
 @import 'responsive';
 @import 'connection-status';
+@import "hd-video-alert";
```
Comment (hollandmatt) [inline css/main.scss:106]: Just for consistency:

```suggestion
@import 'hd-video-alert';
```

**Review by hollandmatt — APPROVED** [review]
👍  Looks good, just found a few typos!

**Comment by jngo-janeapp** [comment]
## Before Testing
- [x] CI must be passing
- [x] No conflicts
- [x] No unresolved reviews/changes requested
- [x] No additional changes/pushes from dev since the last round of QA
- [x] Release note is sufficient for [#features](https://janeapp.slack.com/archives/C0KC1395G)
- [x] Read through Notion / Jira story / Linear Issue / Slack Convo / Demo video
- [x] Read through comments

## Developer's Recommended Test Plan
- [x] Telehealth Selectable HD quality beta flag is enabled > 
Demo_Clinic_-_Jane_Clinic_Management
- [x] Create an online appointment > 
Schedule_-_Demo_Clinic_-_Jane_and_My_Account___Demo_Clinic_and_WhatsApp
- [x] Join the call > wait until the Load HD for better video quality? alert pops up. (if the connection is "good" & the bandwidth > 2000) > click on the Improve my video quality button > the video feed should change to HD(720) at full screen.
7_Df_6_F_3_A_9___Jane_Video_Chat_and_Chart_-_Patients_-_Demo_Clinic_-_Jane

## Additional Thoughts on Testing
- [x] Ensure all buttons work as expected
7_Df_6_F_3_A_9___Jane_Video_Chat_and_Chart_-_Patients_-_Demo_Clinic_-_Jane_and_WhatsApp
 
## Mobile, Browsers, Jane Desktop
- [x] Google Chrome
- [x] Firefox
- [x] Safari
- [x] iPhone

## Pass Story on QA Board / Mark Done in Jira/ Mark Scheduled for Release in Linear
- [x] LGTM

## Move PR on Roll Out Board
- [x] Moved from **QA** to **Patch Release Candidate**

**Comment by jngo-janeapp** [comment]
This PR is ✅

**Pull request merged**